docker/dockerfiles/vflow-node.Dockerfile

```diff
@@ -63,8 +64,7 @@ RUN apt-get update -qq \
     && rm -rf /var/{lib/apt/lists/*,cache/apt/archives/*.deb} /tmp/*
 
 COPY --from=builder "/usr/src/node/target/${PROFILE}/${BINARY}" "/usr/local/bin/"
-COPY --from=builder "/usr/src/node/target/${PROFILE}/wbuild/vflow-volta-runtime/vflow_volta_runtime.compact.compressed.wasm" "./vflow_volta_runtime.compact.compressed.wasm"
-COPY --from=builder "/usr/src/node/target/${PROFILE}/wbuild/vflow-mainnet-runtime/vflow_mainnet_runtime.compact.compressed.wasm" "./vflow_mainnet_runtime.compact.compressed.wasm"
```

Comment:
Remember to remove also this copy... CI will don't catch it till we'll do the next release.

Reply:
You mean to remove also the `Volta` runtime? I thought we decided to leave it there